### PR TITLE
quota: upgrade to 4.09

### PR DESCRIPTION
**Pull request by miz060**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?

###### Change Log  

- Upgrade `quota` to 4.09
- Remove unnecessary patches

###### Does this affect the toolchain?  

**NO**

###### Test Methodology

- BuddyBuild: [499110](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=499110&view=results)

### CODE REVIEW

File: SPECS/quota/quota.signatures.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
  "Signatures": {
-  "quota-4.06.tar.gz": "2f3e03039f378d4f0d97acdb49daf581dcaad64d2e1ddf129495fd579fbd268d",
+  "quota-4.09.tar.gz": "9cdaca154bc92afc3117f0e5f5b3208dd5f84583af1cf061c39baa0a2bb142f9",
   "quota_nld.service": "95235d6fe245ffe3802956036210ca30d5ef5544e1537df9c536dc0f41e6aa95",
   "quota_nld.sysconfig": "3947079eed442d61d5c3447c9908cc672d600940dbf8f8dd2bffee6c2f5967eb",
   "rpc-rquotad.service": "b02f388201740d1736be0e6fb92efa9afce027c5887b0da0b9f710d05df73648",
   "rpc-rquotad.sysconfig": "e4ab26f4608788e6581d58b52d6355792fafe55ecbefc83abb652904c52bcd5d"
  }
-}
+}
```
Comment (mfrw) [inline SPECS/quota/quota.signatures.json:9]: EOF Linefeed
```suggestion
}

```

File: cgmanifest.json [hunk 2]
```diff
@@ -25004,8 +25004,8 @@
         "type": "other",
         "other": {
           "name": "quota",
-          "version": "4.06",
-          "downloadUrl": "http://downloads.sourceforge.net/linuxquota/quota-4.06.tar.gz"
+          "version": "4.09",
+          "downloadUrl": "http://downloads.sourceforge.net/linuxquota/quota-4.09.tar.gz"
```
Comment (mfrw) [inline cgmanifest.json:25008]: Is there a no `https` download link ?

Comment (anphel31) [reply cgmanifest.json:25008]: You could use the following: `https://downloads.sourceforge.net/project/linuxquota/quota-tools/4.09/quota-4.09.tar.gz`

File: SPECS/quota/quota.spec [hunk 3]
```diff
@@ -60,22 +60,6 @@ Source4:        rpc-rquotad.sysconfig
 Patch0:         quota-4.06-warnquota-configuration-tunes.patch
```
Comment (anphel31) [inline SPECS/quota/quota.spec:60]: Can we switch the `URL:` to https as well? `https://sourceforge.net/projects/linuxquota/`

**Review by anphel31 — APPROVED** [review]

File: SPECS/quota/quota.signatures.json (hunk 1 above)
Comment (miz060) [reply SPECS/quota/quota.signatures.json]: EOF line added.

File: SPECS/quota/quota.spec (hunk 3 above)
Comment (miz060) [reply SPECS/quota/quota.spec:60]: Good point!! URL updated to https.

File: cgmanifest.json (hunk 2 above)
Comment (miz060) [reply cgmanifest.json]: Good point!! URL updated to https.

**Pull request merged**